### PR TITLE
Add Heroku section under Services & Integrations in the docs.

### PR DESCRIPTION
**Pull request by matsimitsu**
While writing documentation for the new Heroku features I had great difficulties picking the right place to put these docs.

Since we have quite a few Heroku-specific features I think it's good to give it a prominent place in the navigation.

* Move host metrics to Heroku section (added Netlify redirect to not break the old link).
* Add Heroku section with:
* * Logdrain
* * Errors
* * Host metrics
* * Dashboards

Screenshot 2021-05-05 at 12 57 00

### CODE REVIEW

**Review by shairyar — APPROVED** [review]

**Comment by roytomeij** [comment]
Would it make sense to also move the [Deploy Markers docs for Heroku](https://docs.appsignal.com/application/markers/deploy-markers.html#heroku-support) to here?

PS: We also have [documentation in Heroku's Dev Center](https://devcenter.heroku.com/articles/appsignal). Keeping info shared across two platforms will probably lead to a hot mess. I'm looking into this.

**Comment by matsimitsu** [comment]
> Would it make sense to also move the Deploy Markers docs for Heroku section here?

Ah yeah, I'll add that here too!

File: source/heroku/dashboards.html.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+---
+title: "Heroku dashboards"
+description: "Learn how to set up Heroku Dashboards and what each dashboard means."
+---
+
+-> **Tip:** Make sure a working [Heroku Logplex Logdrain](/herou/setup-logdrain.html) is setup before continuing with this section.
```
Comment (tombruijn) [inline source/heroku/dashboards.html.md:6]:
```suggestion
-> **Tip:** Make sure a working [Heroku Logplex Logdrain](/heroku/setup-logdrain.html) is setup before continuing with this section.
```

File: source/heroku/logplex-errors.html.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+---
+title: "Heroku Logplex Errors"
+description: "Learn how to set up Heroku Logplex Errors what the errors mean."
+---
+
+-> **Tip:** Make sure a working [Heroku Logplex Logdrain](/herou/setup-logdrain.html) is setup before continuing with this section.
```
Comment (tombruijn) [inline source/heroku/logplex-errors.html.md:6]:
```suggestion
-> **Tip:** Make sure a working [Heroku Logplex Logdrain](/heroku/setup-logdrain.html) is setup before continuing with this section.
```

File: source/heroku/host-metrics.html.md [hunk 3]
```diff
@@ -1,9 +1,9 @@
 ---
-title: "Heroku host metrics <sup>beta</sup>"
+title: "Heroku host metrics"
 description: "Learn how to set up host metrics for Heroku dyno's for AppSignal using Heroku's logplex system."
 ---
 
--> **Note**: This page is part of the [host metrics section](/metrics/host.html).
+-> **Tip:** Make sure a working [Heroku Logplex Logdrain](/herou/setup-logdrain.html) is setup before continuing with this section.
```
Comment (tombruijn) [inline source/heroku/host-metrics.html.md:6]:
```suggestion
-> **Tip:** Make sure a working [Heroku Logplex Logdrain](/heroku/setup-logdrain.html) is setup before continuing with this section.
```

**Review by tombruijn — APPROVED** [review]

**Pull request merged**